CRITICAL - 2021-11-27 07:07:22 --> unlink(img/1608368239_2ded8efc6af0e07ea1d5.jpg): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(img/1608...', 'D:\\xampp\\htdocs...', 115)
#1 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(115): unlink('img/1608368239_...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->delete('4')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-11-27 07:10:31 --> unlink(img/1608368239_2ded8efc6af0e07ea1d5.jpg): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(img/1608...', 'D:\\xampp\\htdocs...', 115)
#1 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(115): unlink('img/1608368239_...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->delete('4')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-11-27 07:13:43 --> unlink(img/1608368239_2ded8efc6af0e07ea1d5.jpg): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(img/1608...', 'D:\\xampp\\htdocs...', 115)
#1 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(115): unlink('img/1608368239_...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->delete('4')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#7 {main}
ERROR - 2021-11-27 20:54:55 --> Unknown column 'aspek_pelapor' in 'field list'
CRITICAL - 2021-11-27 20:54:55 --> Unknown column 'aspek_pelapor' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:02:33 --> Unknown column 'aspek_pelapor' in 'field list'
CRITICAL - 2021-11-27 21:02:33 --> Unknown column 'aspek_pelapor' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:04:57 --> Unknown column 'aspek_pelapor' in 'field list'
CRITICAL - 2021-11-27 21:04:57 --> Unknown column 'aspek_pelapor' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:13:47 --> Unknown column 'aspek_pelapor' in 'field list'
CRITICAL - 2021-11-27 21:13:47 --> Unknown column 'aspek_pelapor' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:18:38 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2021-11-27 21:18:38 --> Unknown column 'created_at' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:22:02 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2021-11-27 21:22:02 --> Unknown column 'created_at' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:22:05 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2021-11-27 21:22:05 --> Unknown column 'created_at' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-11-27 21:25:43 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2021-11-27 21:25:43 --> Unknown column 'created_at' in 'field list'
#0 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `la...', 0)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `la...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `la...')
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1759): CodeIgniter\Database\BaseConnection->query('INSERT INTO `la...', Array, false)
#4 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, false)
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(105): CodeIgniter\BaseModel->save(Array)
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->save()
#10 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#11 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2021-11-27 16:48:03 --> Undefined array key "waktu_dibuat"
#0 D:\xampp\htdocs\coba\app\Views\lapor\index.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'D:\\xampp\\htdocs...', 35)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('lapor/index', Array, true)
#4 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(45): view('lapor/index', Array)
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->index()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-11-27 16:48:41 --> Undefined array key "waktu_dibuat"
#0 D:\xampp\htdocs\coba\app\Views\lapor\index.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'D:\\xampp\\htdocs...', 35)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('lapor/index', Array, true)
#4 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(45): view('lapor/index', Array)
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->index()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-11-27 16:48:42 --> Undefined array key "waktu_dibuat"
#0 D:\xampp\htdocs\coba\app\Views\lapor\index.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'D:\\xampp\\htdocs...', 35)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('lapor/index', Array, true)
#4 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(45): view('lapor/index', Array)
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->index()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-11-27 16:48:43 --> Undefined array key "waktu_dibuat"
#0 D:\xampp\htdocs\coba\app\Views\lapor\index.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'D:\\xampp\\htdocs...', 35)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('lapor/index', Array, true)
#4 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(45): view('lapor/index', Array)
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->index()
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-11-27 16:49:00 --> Undefined array key "waktu_dibuat"
#0 D:\xampp\htdocs\coba\app\Views\lapor\detail.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'D:\\xampp\\htdocs...', 31)
#1 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Common.php(1097): CodeIgniter\View\View->render('lapor/detail', Array, true)
#4 D:\xampp\htdocs\coba\app\Controllers\Lapor.php(59): view('lapor/detail', Array)
#5 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(824): App\Controllers\Lapor->detail('6')
#6 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Lapor))
#7 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\coba\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\xampp\htdocs\coba\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\xampp\\htdocs...')
#10 {main}
